x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/machine_learning_rule_suppression.cy.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getMachineLearningRule } from '../../../../objects/rule';
+import { TOOLTIP } from '../../../../screens/common';
+import {
+  ALERT_SUPPRESSION_FIELDS,
+  ALERT_SUPPRESSION_FIELDS_INPUT,
+} from '../../../../screens/create_new_rule';
+import {
+  DEFINITION_DETAILS,
+  DETAILS_TITLE,
+  SUPPRESS_BY_DETAILS,
+  SUPPRESS_FOR_DETAILS,
+  SUPPRESS_MISSING_FIELD,
+} from '../../../../screens/rule_details';
+import {
+  executeSetupModuleRequest,
+  forceStartDatafeeds,
+  forceStopAndCloseJob,
+} from '../../../../support/machine_learning';
+import {
+  continueFromDefineStep,
+  fillAlertSuppressionFields,
+  fillDefineMachineLearningRule,
+  selectMachineLearningRuleType,
+  selectAlertSuppressionPerInterval,
+  setAlertSuppressionDuration,
+  selectDoNotSuppressForMissingFields,
+  skipScheduleRuleAction,
+  fillAboutRuleMinimumAndContinue,
+  createRuleWithoutEnabling,
+} from '../../../../tasks/create_new_rule';
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { getDetails } from '../../../../tasks/rule_details';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+
+describe(
+  'Machine Learning Detection Rules - Creation',
+  {
+    tags: ['@ess', '@serverless', '@skipInServerlessMKI'],
```

Comment:
can we add a reson why it skipped in MKI? I assume it's a feature flag?

Reply:
Yes, this was in response to [your previous comment](https://github.com/elastic/kibana/pull/181926#discussion_r1634541434)

Reply:
comment, that explains why it is skipped, can be useful for other devs, who are not aware of why it was placed there in the first place

Reply:
Yes, I usually let [the commit message](240e9981b69be69ec7530999e34d87c6ebe8d943) serve that purpose, but I _still_ have not adapted to the fact that our squashing strategy makes that mostly useless. I will add a comment.